Include library to compile properly under sparc platformIndex: lshw-02.14/src/core/abi.cc
===================================================================
--- a/src/core/abi.cc
+++ b/src/core/abi.cc
@@ -11,6 +11,7 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <dirent.h>
+#include <stdlib.h>
 
 __ID("@(#) $Id: mem.cc 1352 2006-05-27 23:54:13Z ezix $");
 
